pylint/__pkginfo__.py

```diff
@@ -2,37 +2,42 @@
 # For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
 # Copyright (c) https://github.com/PyCQA/pylint/blob/main/CONTRIBUTORS.txt
 
+"""This module exists for compatibility reasons.
+
+It's updated via tbump, do not modify.
+"""
+
 from __future__ import annotations
 
 __version__ = "2.14.0-dev0"
 
 
-def get_numversion_from_version(v: str) -> tuple[int, int, int]:
+def get_numversion_from_version(version: str) -> tuple[int, int, int]:
```

Comment:
I don't think we can change the signature like that. `get_numversion_from_version(v="test")` won't work anymore.

Reply:
Sure, but I think it's internal, it was introduced in 2.8.2 and our version and tag are clean semver at this point.

Reply:
Let's be conservative. With all the deprecations and changes in `2.14` we are already at risk of breaking compatibility somewhere in `2.14`... 😅

Reply:
87f2cf032